content/building/environment-variables.md

```diff
@@ -7,9 +7,14 @@ weight: 2
 Environment variables are useful for storing information that you do not want to store in the repository, such as your credentials or workflow-specific data. In addition, you can make use of a number of read-only environment variables that Codemagic exports to customize your builds.
 
 {{<notebox>}}
-You can add environment variables in the [environment](../getting-started/yaml/#environment) section of the `codemagic.yaml` configuration file or in the [Environment variables](../flutter/env-variables/) section in the Flutter workflow editor.
+You can add environment variables in the following places:
 
-See how to [encrypt sensitive information](./encrypting) in Codemagic.
+* [environment](../getting-started/yaml/#environment) section of the `codemagic.yaml` configuration file
+* [Environment variables](../flutter/env-variables/) section in the Flutter workflow editor
+* [Global variables and secrets](.../building/environment-variable-groups/#global-variables-and-secrets) of the team settings page
```

Comment:
I guess team settings should be with capital letter as well, if Application settings is